tests/utils.go

```diff
@@ -2063,7 +2063,19 @@ func NewRandomFedora32VMIWithFedoraUser() *v1.VirtualMachineInstance {
 }
 
 func NewRandomFedoraVMIWitGuestAgent() *v1.VirtualMachineInstance {
-	agentVMI := NewRandomVMIWithEphemeralDiskAndUserdata(ContainerDiskFor(ContainerDiskFedora), GetGuestAgentUserData())
+	virtClient, err := kubecli.GetKubevirtClient()
+	PanicOnError(err)
+
+	dnsServerIP, err := getClusterDnsServiceIP(virtClient)
+	PanicOnError(err)
+
+	networkData := ""
+	if netutils.IsIPv6String(dnsServerIP) {
```

Comment:
Can we set the IPv6 configuration even for IPv4 setups?
In the end, the pod IP determines which test scenario should exist, preparing the VM with the same config has the advantage of making it ready for dual stack.

Reply:
We can, but I honestly don't see it as an advantage - a user reading the tests will be mislead into thinking we support dual stack.

Once we do support it, we should update the code with the change you're proposing, IMHO.

Reply:
The advantage is not to have the code above it and ask the question based on its result.
You can add a comment if that is helpful to make the point clear here.

BTW, the VM **is** dual stack (most likely).

Reply:
cloud-init creates ifcfg files, then does an ifup which NM overtakes.

Reply:
I meant that in an ideal world, NM would start *after* cloud-init is done. 

But anyway, looks promising, locally, the tests I've ran worked fine. Let's wait for CIs verdict.